frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewSubformLayoutSet/CreateNewSubformLayoutSet.tsx

```diff
@@ -29,6 +34,7 @@ export const CreateNewSubformLayoutSet = ({
     });
```

Comment:
The value of the `selectedOption` doesn't seem to be used. Shouldn't we add it to the layoutSet? Something like:

```
addLayoutSet({
      layoutSetIdToUpdate: newSubform,
      layoutSetConfig: {
        id: newSubform,
        type: 'subform',
        datatype: selectedOption?.value...
      },
    });
```

Reply:
I added it

Reply:
I have fixed merge conflict now and i see that this part of code is moved to seperate hooks , so the datatype is not added yet, because it should be fixed in the new hook called `useCreateSubform`

Reply:
I see that you used the `fileName` for the `dataType`, but I wonder if it shouldn't rather be something like the datamodel ID, as we did in the process editor. 🤔

Reply:
Here is a possible solution reusing the code from the process editor: https://github.com/Altinn/altinn-studio/commit/e3707da38fef335639cce351e6f4acf7e47fd4b2